packages/xo-server/src/xapi/mixins/pool.js

```diff
@@ -0,0 +1,18 @@
+import { cancelable } from 'promise-toolbox'
+
+import { parseXml } from '../../utils'
+
+export default {
+  @cancelable
+  getPoolMetadata($cancelToken) {
+    const { pool } = this
+    return this.getResource($cancelToken, '/pool/xmldbdump', {
+      task: this.createTask(
+        'Pool metadata',
+        pool.name_label ?? pool.$master.name_label
+      ),
+    })
+      .then(response => response.readAll())
+      .then(parseXml)
```

Comment:
No need to parse it, for now we won't care about the format, we'll just export it and import it back in the XenAPI.